src/index.js

```diff
@@ -35,6 +34,7 @@ export {default as NavigationControl} from './components/navigation-control';
 
 // Utilities
 // export {default as fitBounds} from './utils/fit-bounds';
+export {default as MapControls} from './utils/map-controls';
```

Comment:
There was an intention not to export the new internal event management so that we could have a bit more time to let the new event handling APIs "bake" before we freeze them.

Any strong reason to include this in the export list? Is this export necessary to use `react-map-gl`?

Without this export, I's be happy to approve this PR pretty much as is. With this line in place, we should have at least add a documentation page for the MapControls class and I'd want to take a more thorough look at the API.